infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 domain     = "registry.k8s.io"
 project_id = "k8s-infra-oci-proxy-prod"
-tag        = "v20221005-v0.0.2-28-g52c8633"
+tag        = "v0.0.4"
```

Comment:
let's pin the digest this time.
small bit of defense in depth.

Reply:
Fixed.